cmd/loadTests.go

```diff
@@ -1227,6 +1241,20 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 					}
 					if pipelineRun.IsDone() {
 						succeededCondition := pipelineRun.Status.GetCondition(apis.ConditionSucceeded)
+                        pvcs, err := framework.AsKubeAdmin.TektonController.KubeInterface().CoreV1().PersistentVolumeClaims(pipelineRun.Namespace).List(context.Background(), metav1.ListOptions{})
+                            if err != nil {
+                                fmt.Sprintf("Error getting PVC: %v\n", err)
+                            }
+                            for _, pvc := range pvcs.Items {
+                                pv, err := framework.AsKubeAdmin.TektonController.KubeInterface().CoreV1().PersistentVolumes().Get(context.Background(), pvc.Spec.VolumeName, metav1.GetOptions{})
+                                if err != nil {
+                                    fmt.Sprintf("Error getting PV: %v\n", err)
+                                    continue
+                                }
+                                waittime := (pv.ObjectMeta.CreationTimestamp.Time).Sub(pvc.ObjectMeta.CreationTimestamp.Time)
+                                PipelineRunWaitTimeForPVCSumPerThread[threadIndex] += waittime
+                                SuccessfulPVCCreationsPerThread[threadIndex] += 1
+                            }
```

Comment:
```suggestion
pvcs, err := framework.AsKubeAdmin.TektonController.KubeInterface().CoreV1().PersistentVolumeClaims(pipelineRun.Namespace).List(context.Background(), metav1.ListOptions{})
						if err != nil {
							logError(23, fmt.Sprintf("Error getting PVC: %v\n", err))
						}
						for _, pvc := range pvcs.Items {
							pv, err := framework.AsKubeAdmin.TektonController.KubeInterface().CoreV1().PersistentVolumes().Get(context.Background(), pvc.Spec.VolumeName, metav1.GetOptions{})
							if err != nil {
								logError(24, fmt.Sprintf("Error getting PV: %v\n", err))
								continue
							}
							waittime := (pv.ObjectMeta.CreationTimestamp.Time).Sub(pvc.ObjectMeta.CreationTimestamp.Time)
							PipelineRunWaitTimeForPVCSumPerThread[threadIndex] += waittime
							SuccessfulPVCCreationsPerThread[threadIndex] += 1
						}
```

Reply:
@siddardh-ra The above suggestion fixes golint and also the indentation, PTal

Reply:
Yes, Thanks @pmacik. Updated the code as per the suggestion :+1: